app/views/admins/show_pending_bills.html.erb

```diff
@@ -0,0 +1,13 @@
+Pending Bills | <%= link_to "Back To Admin Page", admins_path() %>
```

Comment:
You don't need to add empty parentheses to your `index` path.

Reply:
removed ().